hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/src/main/java/com/google/cloud/bigtable/hbase2_x/replication/HbaseToCloudBigtableReplicationEndpoint.java

```diff
@@ -79,7 +79,7 @@ public void init(Context context) throws IOException {
 
   @Override
   public boolean canReplicateToSameCluster() {
-    return false;
+    return true;
```

Comment:
Is this intended?

Reply:
yes, this was a bug that started getting caught via a new check in 2.4.6. Worked on this with @vermas2012